Library/Tests/ViewModels/ProjectCreatorViewModelTests.swift

```diff
@@ -59,44 +61,47 @@ final class ProjectCreatorViewModelTests: TestCase {
 
     self.goToSafariBrowser.assertValues([])
 
-    self.vm.inputs.decidePolicy(
+    var policy = self.vm.inputs.decidePolicy(
       forNavigationAction: MockNavigationAction(
-        navigationType: .Other,
-        request: .init(URL: NSURL(string: "https://www.kickstarter.com/projects/a/b/creator_bio")!)
+        navigationType: .other,
+        request: .init(url: URL(string: "https://www.kickstarter.com/projects/a/b/creator_bio")!)
       )
     )
+    XCTAssertEqual(.allow, policy)
 
     self.goToSafariBrowser.assertValues([])
 
-    self.vm.inputs.decidePolicy(
+    policy = self.vm.inputs.decidePolicy(
       forNavigationAction: MockNavigationAction(
-        navigationType: .LinkActivated,
-        request: .init(URL: NSURL(string: "http://www.google.com")!)
+        navigationType: .linkActivated,
+        request: .init(url: URL(string: "http://www.google.com")!)
       )
     )
+    XCTAssertEqual(.cancel, policy)
```

Comment:
swift 3 caught some unused values in these tests so i went ahead and did asserts on them. make these tests a lil stronger